econet_heatpump_water_heater.yaml

```diff
@@ -10,6 +10,9 @@ substitutions:
   external_components_source: github://esphome-econet/esphome-econet@main
   logger_level: WARN
   econet_update_interval: 30s
+  # The following has been found to give more accurate results but is likely device specific
+  # sensor_power_filters_lambda: return x * 0.91 - 20;
```

Comment:
Should this actually move to the example-esp32.yaml? Like, if this is intended to be something that an individual user opts into wouldn't it belong there?

Reply:
I don't expect most use users to be able to get actual measurements to be able to adjust this. In the README we already say "See the individual device configuration files for more customizeable options."

Reply:
Sure, I think I agree there. I guess my point is that either:
* We think this is right and we should make it default behavior with some comment detailing how we came up with it, or
* We think this is an advanced tweak in which case put it in your own settings or call it out on the wiki but don't include it in the settings file.

Reply:
This is definitely an advanced tweak. The problem is it cannot be put in my own settings without having the substitutions here. I would have to stop using the package that I'd rather not. Are you suggesting dropping the comment or the whole PR?

Reply:
Oh, I see. If you don't include the substitution it can't be over-ridden. In that case I think it's fine to include as it.